Tasks/VsTest/VSTest.ps1

```diff
@@ -10,15 +10,82 @@ param(
     [string]$testRunTitle,
     [string]$platform,
     [string]$configuration,
-    [string]$publishRunAttachments
+    [string]$publishRunAttachments,
+    [string]$runInParallel
 )
 
     
-Function CmdletHasMember($memberName) {
+function CmdletHasMember($memberName) {
     $publishParameters = (gcm Publish-TestResults).Parameters.Keys.Contains($memberName) 
     return $publishParameters
 }
 
+function SetRegistryKeyForParallel($vsTestVersion) {
```

Comment:
it is set only when VS is launched. adding this in to avoid asking users to launch VS on remote BA machine before using this task